src/browser/provider/phantomjs.js

```diff
@@ -0,0 +1,47 @@
+import { create as createPhantom } from 'phantom';
+
+export default {
```

Comment:
It will show nothing, because default implementation of `getAvailableAliases` returns `[].` Maybe it's better to show a descriptive message here, like "The provider "{providerName}" doesn't support browser aliases, just use "${providerName}" when specifying browsers"?